kvm.rb

```diff
@@ -2,8 +2,10 @@
 
 class Kvm < Formula
   homepage "https://www.github.com/aspnet/Home"
-  version "1.0.0-beta1"
-  url "https://github.com/aspnet/Home.git", :branch => 'master'
+  version "1.0.0-beta3"
+  url "https://github.com/aspnet/Home.git", :tag => 'v1.0.0-beta3'
+
+  head "https://github.com/aspnet/Home.git", :branch => 'dev'
```

Comment:
That's pretty uncommon in the Homebrew world. Every formula I could find in the [main repository](https://github.com/Homebrew/homebrew/tree/master/Library/Formula) uses explicit version numbers in the URL of the package to download. The `version` field can be auto-detected from the URL, but it doesn't seem to work with our tag format, and even then we have to explicitly specify the tag. So regardless, we're going to have the version embedded at least once in this file.

At least with this change we get two benefits:

1. Users who want to update to the bleeding edge can use "brew install kvm --HEAD"
2. We only have to update the version number for future updates, no other changes are required.